authentication/src/main/java/io/camunda/authentication/config/WebSecurityConfig.java

```diff
@@ -302,6 +302,13 @@ public JwtDecoder jwtDecoder(
             JwtValidators.createDefaultWithValidators(new AudienceValidator(validAudiences)));
       }
 
+      final var expectedClaims =
+          securityConfiguration.getAuthentication().getOidc().getExpectedClaims();
+      if (expectedClaims != null) {
+        decoder.setJwtValidator(
+            JwtValidators.createDefaultWithValidators(new ClaimValidator(expectedClaims)));
```

Comment:
:x: This would overwrite the audience validator from above. We should construct a composite validator.